synapse/tests/test_lib_hive.py

```diff
@@ -288,7 +313,7 @@ def onedit(mesg):
             with self.raises(s_exc.AuthDeny):
                 await s_hive.openurl(turl, user='root', passwd='newpnewp')
 
-            async with await s_hive.openurl(turl, user='root', passwd='secret') as hive0:
+            async with await s_hive.openurl(turl, user='root', passwd='secret'):
```

Comment:
what condition is this validating??? that a user can connect just fine after unlocking? if so a docstring would be good here for future us.

Reply:
err comment

Reply:
Added comments.